docs/contributing.md

```diff
@@ -58,3 +58,21 @@ externally visible changes to etcd itself. This is to avoid the tests causing th
 For example an end to end test may create an `EtcdCluster` and assert that it can connect to it from inside the cluster
 using the expected DNS name. Elements of the Kuberentes API that a user might interact with, such as the `status` field
 on an `EtcdCluster` resource, may also be interacted with.
+
+## Release Process
+
+Releases of the operator occur whenever the team feel there is sufficient new features to produce a release. The version
+string will be prefixed with `v` and use semver.
```

Comment:
:+1: this works for now, and it's best that it is documented regardless of what that process is. That said, I think longer term, having an i.e. monthly cadence is more obvious to end-users when a release will happen 😄

Reply:
Yeah this is entirely designed to be a 'good enough for now' system.